tests/results/ResultFilterTest.py

```diff
@@ -368,6 +368,288 @@ def test_affected_code(self):
+                                               end_column=5)
+
+        res1_pre_addition = Result(origin="origin",
+                                   message="message",
+                                   affected_code=(sr1_pre_addition,))
+        res1_addition = Result(origin="origin",
+                               message="message",
+                               affected_code=(sr1_addition,))
+        res1_post_addition = Result(origin="origin",
+                                    message="message",
+                                    affected_code=(sr1_post_addition,))
+        res1_around_addition = Result(origin="origin",
+                                      message="message",
+                                      affected_code=(sr1_pre_addition,
+                                                     sr1_post_addition))
+        res1_with_addition = Result(origin="origin",
+                                    message="message",
+                                    affected_code=(sr1_pre_addition,
+                                                   sr1_addition,
+                                                   sr1_post_addition))
+        res1_whole_addition = Result.from_values(origin="origin",
+                                                 message="message",
+                                                 file="file_name_new",
+                                                 line=9,
+                                                 column=1,
+                                                 end_line=11,
+                                                 end_column=5)
+
+        original_result_list = [res0_pre_change,
+                                res0_change,
+                                res0_post_change,
+                                res0_around_change,
+                                res0_with_change,
+                                res0_whole_change,
+
+                                res0_pre_remove,
+                                res0_post_remove,
+                                res0_around_remove,
+                                res0_whole_remove,
+
+                                res0_pre_addition,
+                                res0_post_addition,
+                                res0_around_addition,
+                                res0_whole_addition]
+
+        new_result_list = [res1_pre_change,       # FALSE POSITIVE (in-line)
+                           res1_change,           # correctly kept
+                           res1_post_change,      # FALSE POSITIVE (in-line)
+                           res1_around_change,    # FALSE POSITIVE (in-line)
+                           res1_with_change,      # correctly kept
+                           res1_whole_change,     # correctly kept
+
+                           res1_pre_remove,       # correctly filtered out
+                           res1_post_remove,      # FALSE POSITIVE (in-line)
```

Comment:
ahm did you copy another test? :/ with the false positives? We only want to test renaming here, not everything, right?